internal/server/options.go

```diff
@@ -267,3 +269,10 @@ func WithWorkerPool(workerIds []string) Option {
 		o.withWorkerPool = workerIds
 	}
 }
+
+// WithLocalStorageState provides an optional local storage state.
+func WithLocalStorageState(state string) Option {
```

Comment:
We should add a unit test for this option in the internal/server/options_test.go file.